tools/packager/packager.py

```diff
@@ -36,6 +38,8 @@
 from gbs import GBS
 from metadata import metadata
 
+sys.path.insert(0, 'jsonschema')
```

Comment:
'jsonschema' directory relative to what? Is this meant to facilitate running directly in the git repo?